src/onDraw.js

```diff
@@ -1,5 +1,47 @@
-import updateSubjectCount from './util/update-subject-count';
+import updateSubjectCount
+    from './util/update-subject-count';
 
-export default function onDraw(){
-	updateSubjectCount(this, this.config.id_col, ".annote")
-}
\ No newline at end of file
+export default function onDraw() {
+  //Annotate number of selected participants out of total participants.
+	updateSubjectCount(this, this.config.id_col, '.annote');
+
```

Comment:
Note: Not going to code review the chunk below since it is fairly low priority and also fairly complex. We can always debug (or just take out the functionality) if it becomes problematic.